### PR TITLE
Small error improvement

### PR DESCRIPTION
**Pull request by damienfa**
Here is the small improvement that I proposed the 3th of May.

### CODE REVIEW

**Comment by samzilverberg** [comment]
hey @damienfa sorry for not checking yet, its end of the year and theres as always lots of stuff keeping me busy. I believe I'll have time beginning of 2017 to take a look! 
Merry Christmas/Hanuka and a happy new year

File: src/browser/MixpanelProxy.js [hunk 1]
```diff
@@ -1,10 +1,21 @@
...
+      (console && console.error) && console.error(paramName, value); 
+      return "invalid"; 
+    }
+    if (typeof value !== 'string') {
+      (JSON && JSON.stringify) && (value = JSON.stringify(value));
+    }
     return 'invalid ' + paramName + ': ' + value;
   },
   notsupported: function(methodName) {
+    if (typeof paramName !== 'string') {
```
Comment (samzilverberg) [inline src/browser/MixpanelProxy.js]: type? should be `if (typeof methodName ... ` right?

**Comment by damienfa** [comment]
@samzilverberg, Sure ! My mistake ...

**Comment by samzilverberg** [comment]
merged, but probably won't release soon. 
its too much hassle to release just for a small improvement for errors on browser platform :)

will try to release in next 1 or 2 months along with mixpanel ios/android lib updates

**Pull request merged**